File: AdministrativeService/AdministrativeService.Core/Entities/Shop.cs
namespace AdministrativeService.Core.Entities
{
	/// <summary>
	/// Shop.
	/// </summary>
	public class Shop : BaseEntity
	{
		/// <summary>
		/// Shop's title.
		/// </summary>
		public required string Title { get; set; }

		/// <summary>
		/// Shop's description.
		/// </summary>
		public string? Description { get; set; }

		public long? VkGroupId { get; set; }

		public List<ShopAdmin>? Admins { get; set; }
	}
}

File: AdministrativeFront/src/app/contracts/shop.ts
export interface CreateShopRequest {
    title: string;
    description: string | undefined;
}

export interface Shop {
    id: string;
    title: string;
    description: string | undefined;
    vkGroupId: number | null;
    admins: ShopAdmin[];
}

export interface ShopAdmin {
    id: string;
    username: string;
    feature: AdminFeature;
}

export enum AdminFeature {
    CanAll = 15
}